tests/tests_integration/tests_diff_version/conftest.py

```diff
@@ -1,157 +1,129 @@
         )
     )
-
     return case
 
 
 @pytest.fixture
-def demo_get_messages_with_one_filter(demo_data_source: HTTPProviderDataSource) -> Data:
-    case = demo_data_source.command(
+def get_messages_with_one_filter(data_source: HTTPProviderDataSource) -> Data:
+    case = data_source.command(
         http.GetMessages(
-            start_timestamp=datetime(year=2022, month=6, day=30, hour=14, minute=48, second=20, microsecond=0),
-            end_timestamp=datetime(year=2022, month=6, day=30, hour=14, minute=48, second=25, microsecond=0),
-            stream=["arfq01fix07"],
-            filters=Filter("type", "NewOrderSingle"),
+            start_timestamp=START_TIME,
+            end_timestamp=END_TIME,
+            stream=["ds-lib-session1","ds-lib-session2"],
+            filters=Filter("body", MESSAGE_ID_1),
         )
     )
 
     return case
 
 
 @pytest.fixture
-def demo_get_messages_with_filters(demo_data_source: HTTPProviderDataSource) -> Data:
-    case = demo_data_source.command(
+def get_messages_with_filters(data_source: HTTPProviderDataSource) -> Data:
+    case = data_source.command(
         http.GetMessages(
-            start_timestamp=datetime(year=2022, month=6, day=30, hour=14, minute=48, second=20, microsecond=0),
-            end_timestamp=datetime(year=2022, month=6, day=30, hour=14, minute=48, second=25, microsecond=0),
-            stream=["arfq01fix07"],
-            filters=[Filter("type", "NewOrderSingle"), Filter("body", "200")],
-        )
-    )
-
-    return case
-
-
-@pytest.fixture
-def demo_events_from_data_source(demo_data_source: HTTPProviderDataSource) -> Data:
-    events = demo_data_source.command(
-        http.GetEvents(
             start_timestamp=START_TIME,
             end_timestamp=END_TIME,
+            stream=["ds-lib-session1","ds-lib-session2"],
+            filters=[Filter("type", "Incoming"), Filter("body", MESSAGE_ID_1)]
         )
     )
-    # Returns 49 events #TODO
-    # Failed = 6
-    return events
 
-
-@pytest.fixture
-def demo_messages_from_data_source(demo_data_source: HTTPProviderDataSource) -> Data:
-    messages = demo_data_source.command(
-        http.GetMessages(
-            start_timestamp=datetime(year=2022, month=6, day=30, hour=14, minute=58, second=0, microsecond=0),
-            end_timestamp=END_TIME,
-            stream=["arfq01fix07"],
-        )
-    )
-    # Returns 239 messages
-    return messages
-
-
-@pytest.fixture
-def demo_events_from_data_source_with_cache_status(
-    demo_data_source: HTTPProviderDataSource,
-) -> Data:
-    events = demo_data_source.command(http.GetEvents(start_timestamp=START_TIME, end_timestamp=END_TIME, cache=True))
-    # Returns 49 events #TODO
-    # Failed = 6
-    return events
+    return case
 
 
 @pytest.fixture
-def demo_messages_from_data_source_with_test_streams(
-    demo_data_source: HTTPProviderDataSource,
+def events_from_data_source_with_cache_status(
```

Comment:
Previous fixture used lots of streams. It were used for test when user provide many streams. I haven't found any replacement